src/VisualStudio/CSharp/Impl/Options/Formatting/StyleViewModel.cs

```diff
@@ -1157,6 +1157,21 @@ void M2(string[] args)
 //]
     }}
 }}
+";
+
+        private static readonly string s_preferUTF8StringLiterals = $@"
+using System;
+
+//[
+class WebResponse
+{{
+    // {ServicesVSResources.Prefer_colon}
+    private readonly byte[] header = ""Hello""u8;
```

Comment:
should we mention ReadOnlySpan instead?  where are we landing on this wrt to how the lang feature works?  do these make byte[]s?

Reply:
My understanding is the natural type is changing to a ROS<byte> but ROS felt like a bit much to put in an example in VS options, given the audience, but that might be my own preferences leaking through :)